flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is a disappointment.

Reply:
One package in dependent havn't aarch64 version.

Reply:
you mean you don't build it from source? Which one is it?

Reply:
https://pypi.org/project/zxing-cpp/

Reply:
at has a source distribution